client/components/search.jsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+import { getMovies } from '../redux/movies/actions';
+import { movieFilter } from '../utilities';
+import SingleMovieBox from './singleMovieBox';
+
+class Search extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchTerm: '',
+      searchCriteria: 'title',
+      searchedMovies: [],
+    };
+  }
+
+  async componentDidMount() {
+    // eslint-disable-next-line no-shadow
+    const { getMovies } = this.props;
+    await getMovies();
+  }
+
+  handleSearchTermChange(ev) {
```

Comment:
handle searchTermChange and handleSearchCriteriaChange are doing the exact same thing